upstream/CVE-2018-1002102.json

```diff
@@ -0,0 +1,41 @@
+{
+  "id": "CVE-2018-1002102",
+  "modified": "2019-12-03T22:58:37Z",
+  "published": "2019-12-03T22:58:37Z",
+  "summary": "Unvalidated redirect",
+  "details": "Improper validation of URL redirection in the Kubernetes API server in versions prior to v1.14.0 allows an attacker-controlled Kubelet to redirect API server requests from streaming endpoints to arbitrary hosts. Impacted API servers will follow the redirect as a GET request with client-certificate credentials for authenticating to the Kubelet.",
+  "affected": [
+    {
+      "package": {
+        "ecosystem": "kubernetes",
+        "name": "k8s.io/apiserver"
+      },
+      "severity": [
+        {
+          "type": "CVSS_V3",
+          "score": "CVSS:3.1/AV:N/AC:H/PR:H/UI:R/S:C/C:L/I:N/A:N"
+        }
+      ],
+      "ranges": [
+        {
+          "events": [
+            {
+              "introduced": "0"
+            },
+            {
+              "fixed": "1.14.0"
```

Comment:
I am not sure about this value.

Description of CVE says:
> Kubernetes API server in versions prior to v1.14.0

But Issue says:
> Fixed Versions
    Kubernetes v1.14+ - Fixed by default in 
https://github.com/kubernetes/kubernetes/pull/72552
Kubernetes v1.10-v1.14 - Fix available as alpha in
https://github.com/kubernetes/kubernetes/pull/66516

But #66516 has been added to `v.1.13.x` branches - https://github.com/kubernetes/kubernetes/commit/109b67c291de3b9bda35c35e471b9064de6ff859

Also there is cherry-pick to `v1.12` - https://github.com/kubernetes/kubernetes/commit/687c04c53006441ed40f4fa7c09a726cfcec380e

Reply:
```
$ git tag --contains 687c04c53006441ed40f4fa7c09a726cfcec380e | sort -V
v1.12.2
v1.12.3
v1.12.3-beta.0
v1.12.4
v1.12.4-beta.0
v1.12.5
v1.12.5-beta.0
v1.12.6
v1.12.6-beta.0
v1.12.7
v1.12.7-beta.0
v1.12.8
v1.12.8-beta.0
v1.12.9
v1.12.9-beta.0
v1.12.10
v1.12.10-beta.0
v1.12.11-beta.0
```

v1.12.2 seems to be a fixed version.

Reply:
And v1.13.0 has the fix, as you said. So, it would be `{"introduced": "0"}, {"fixed": "1.12.2"}`.

Reply:
Thanks for help!
What do you think of the description?
Should I update it or add a link to this PR in the references?

I think users will have questions about this.

Reply:
>Should I update it or add a link to this PR in the references?

We probably need both 😄 Let's update the description and add a PR link.

Reply:
added in https://github.com/aquasecurity/vuln-list-k8s/pull/62/commits/147f21918ac1c953b58c45b557c8b69499dea977